pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -11,6 +11,7 @@
 import json
 import os
 import re
+import shutil
```

Comment:
Put `shutil` import back below the prelude where it was before. While you're at it could you move `import re` down there too?
```suggestion
```

Reply:
done